app/reducers/index.js

```diff
@@ -2,6 +2,8 @@ import { combineReducers } from 'redux';
 import producerReducer from './producerReducer';
 import consumerReducer from './consumerReducer';
 
+// combines reducers for Producer and Consumer data to so that every reducer is called at once and therefore
```

Comment:
just delete this comment, i think this code is self explanatory and the comment is also a bit misleading